video-wca.html

```diff
@@ -28,12 +28,12 @@
    });
```

Comment:
The changes you've made have commented out the `window.onload` function that sets the video progress to the last recorded position on page load. This means that the video will no longer start at the last recorded position when the page is loaded. 

If you intended to comment out this functionality, it's a valid change. However, if you want to keep the functionality, you should uncomment the code by removing the `//` at the beginning of each line:

```javascript
// Set the video progress to the last recorded position on page load
window.onload = () => {
  const lastPlayTime = localStorage.getItem('lastPlayTime');
  if (lastPlayTime) {
    video.currentTime = lastPlayTime;
  }
};
```

If you are intentionally commenting out the code for some reason (e.g., testing, debugging, or temporarily disabling functionality), make sure to remove the comments if you want the functionality to be active again.